### PR TITLE
implemented Player and child classes

### PR DESCRIPTION
**Pull request by BGBroth**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```
Comment (Dimosphen1) [inline app/main.py:7]: Do not use list comprehension in `sum` as mentioned in the [checklist](https://github.com/mate-academy/py-elves-and-dwarves/blob/master/checklist.md).

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
One minor change was requested regarding list comprehension.

File: app/players/elves/elf_ranger.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return f"Elf ranger {self.nickname}. " \
```
Comment (pavlejviki) [inline app/players/elves/elf_ranger.py:15]: Preferred to use parenthesis instead of \

File: app/players/elves/druid.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        return f"Druid {self.nickname}. {self.nickname} " \
```
Comment (pavlejviki) [inline app/players/elves/druid.py:15]: Preferred to use parenthesis instead of \

**Review by pavlejviki — APPROVED** [review]
Good job, just minor changes are preferable

**Pull request closed without merging**